NEWS

```diff
@@ -11,6 +11,241 @@
 ### Features:
 ### Bugfixes:
 
+## 1.18.0-rc1 (November 21, 2024)
+### Feature
+#### Architecture
+ * Added CPU Model for MI300A
+ * Added Fujitsu ARM specific values to ucx.conf
+#### AZP
+ * Improved repository setup when running CI
+ * Improved test_fuzzy reporting
+ * Updated to CUDA version 12.5.1 for Fabric memory tests
+ * Improved active IB HCA detection on CI
+ * Added DRP release pipeline
+ * Removed io_demo tests for ConnectX-4
+ * Added JUCX tests on ARM
+ * Improved verification and release processes
+ * Added wire-compat testing with v1.17.x
+ * Improved reports with added server-side log prints
+ * Updated AZP images from private repository
+#### Build
+ * Improved compiler error reporting with added flag
+ * Improved coverity script to allow faster turnaround time
+ * Improved Intel Compiler detection and support
+#### Configuration
+ * Implemented Grace config section filtered by CPU model
+#### Continuous Integration
+ * Improved spell checking on generated files
+ * Improved CI by making sure ASAN is actually loaded
+ * Added codespell checking job
+ * Added BlueField ASAN checking job
+#### CUDA
+ * Improved latency estimations on AMD Milan systems
+ * Improved reliability by detecting runtime/build GDR library version match
+ * Added handling missing IPC capability when unpacking keys
+ * Added caching for CUDA IPC memory pool import operation
+ * Added setting platform specific perf estimations with environment variables
+ * Added CUDA Fabric memory support
+ * Improved peer accessible cache for CUDA IPC
```

Comment:
what is this improvement?

Reply:
https://github.com/openucx/ucx/pull/9978, I think it is eventually an improvement, reworded: ".. to be reentrant".